### DISCUSSION TITLE
how can i get minio erause code set size?

### DISCUSSION THREAD

**Discussion by lddlww** [minio/minio#14304] [Q&A]
how can i get minio erasure code set size with ```mcli``` and how can i get total available size in minio？

**Comment by klauspost** [comment]
`mc admin info --json myminio`. Look for info/backend/standardSCParity.

**Reply by lddlww** [reply]
thanks for reply，however i can't find per ```erasure set``` size

**Reply by klauspost** [reply]
Oh, yeah, sorry. Find max `"disk_index": X`, set size is X+1.

**Reply by lddlww** [reply]
thanks，i use  follow command to get max ```disk_index```：
```
mc admin info --json myminio | jq '[.info.servers|.[]|.drives|.[]|.disk_index]|max'
```

**Comment by CaptainLoop** [comment]
I know it's an older question but never was closed, and google already put this on the first result I want to continue this discussion, and not opening a new one, for it's stay on the top in the search. It has valuably answer. 

I'm not exatly understanding the output of `mc admin info --json myminio` What does the parameters means? 

- standardSCParity
- rrSCParity
- set_index
- disk_index

Is this setup good? before I lost all my data because I wrongly configured my minio deployment. 

My setup is 4node 8drive/node. 1.2TB/drive

 I got this output 

		"backendType": "Erasure",
			"onlineDisks": 32,
			"offlineDisks": 0,
			"standardSCParity": 4,
			"rrSCParity": 1,
			"totalSets": [2],
			"totalDrivesPerSet": [16]

`set_index`: half of the drive is 0 other half is 1
```
node1 disk1	disk_index: 0
node1 disk2	disk_index: 4
node1 disk3	disk_index: 8
node1 disk4	disk_index: 12
node1 disk5	disk_index: 0
node1 disk6	disk_index: 4
node1 disk7	disk_index: 8
node1 disk8	disk_index: 12
node2 disk1	disk_index: 2
node2 disk2	disk_index: 6
node2 disk3	disk_index: 10
node2 disk4	disk_index: 14
node2 disk5	disk_index: 2
node2 disk6	disk_index: 6
node2 disk7	disk_index: 10
node2 disk8	disk_index: 14
node3 disk1	disk_index: 3
node3 disk2	disk_index: 7
node3 disk3	disk_index: 11
node3 disk4	disk_index: 15
node3 disk5	disk_index: 3
node3 disk6	disk_index: 7
node3 disk7	disk_index: 11
node3 disk8	disk_index: 15
node4 disk1	disk_index: 1
node4 disk2	disk_index: 5
node4 disk3	disk_index: 9
node4 disk4	disk_index: 13
node4 disk5	disk_index: 1
node4 disk6	disk_index: 5
node4 disk7	disk_index: 9
node4 disk8	disk_index: 13

```
